Documentation/rendering.md

```diff
@@ -1,107 +1,74 @@
+## Camera Controls
 
-To refresh the blob list, for example after a model is converted, press the "Refresh" button on the top.
+Press the **Camera...** button to open a dialog where you can configure the field-of-view, near and far plane, and camera movement speed. The dialog also displays how to control the camera.
 
-## 3. Visualize the model
+## Scene Tree
 
-![Rendering View](media/renderingview.png)
-If the model is loaded correctly, the model will be shown in a new view. On the top you can see from left to right:
+The scene tree on the left displays the structure of the loaded models. You can load multiple models simultaneously. Use the **Remove Models...** button at the top to clear the entire scene. This is especially necessary, if the loaded models exceed the [limits of the rendering VM](https://docs.microsoft.com/azure/remote-rendering/reference/limits#overall-number-of-polygons). In that case you will only see a **checkerboard pattern**.
 
-* The name of the loaded model.
-* "Unload model" button to unload the model and return to the loading page.
-* "Auto-rotate" button to toggle the automatic rotation of the model.
+Select an item either through the tree or by clicking on it in the viewport. If you **double click** an item in the scene tree, the camera will focus on it. The same can be achieved by pressing the **F key** while the viewport has input focus.
 
-The view has a viewport in the center, and three panels around it.
+## Model Scale
 
-### Viewport
+Some models are very large, others are very small and yet others are very far away from the origin. ARRT attempts to detect extreme cases and notify you about it. In any case, you can always use the **Model Scale** option from the toolbar to adjust the size of the loaded models. Also change the camera movement speed as necessary and make sure the near and far plane are configured accordingly.
 
-The viewport shows the rendered model, streaming it from the ARR service. The camera controls are:
+## Material Editing
 
-* WASD keys or arrow keys to move the camera on the horizontal plane
-* QE keys, +- keys, or Page Up and Page Down keys to move the camera vertically
-* Right mouse button, 4682 keys, Insert Home Delete End keys, or ZXFC keys to rotate the camera
-* SHIFT modifier to increase the camera movement speed by a factor of 10
+When an object is selected, the list on the right hand side shows all available the materials. You can edit any material and see the effect live in the viewport. For more information on materials [have a look here](https://docs.microsoft.com/azure/remote-rendering/concepts/materials). Note that there are two types of materials: [PBR materials](https://docs.microsoft.com/azure/remote-rendering/overview/features/pbr-materials) and [color materials](https://docs.microsoft.com/azure/remote-rendering/overview/features/color-materials), which is why you will see different options in the UI depending on the selected material type.
```

Comment:
docs policies discourage you from using "have a look here" as link text. Rather use something like:
More information can be found in the [materials sections](https://docs.microsoft.com/azure/remote-rendering/concepts/materials)